src/context/index.ts

```diff
@@ -326,6 +327,21 @@ export class Context {
     return invalidDids;
   }
 
+  /**
+   * Retrieve information for the current network
+   */
+  public getNetworkProperties(): { name: string; version: BigNumber } {
```

Comment:
Return type should be defined in types as `NetworkProperties`. Version should be a number

There are multiple names apparently. You have `specName`, `implName` and the chain name you get by calling `api.rpc.system.chain`. They are not the same. For example, in pme, you have

```
specName = polymesh
implName = polymath-polymesh
chain = Development (in aldebaran this is Polymesh Aldebaran Testnet)
```

IMO the `chain` value is more significant, so we should return that